yourgov/pages/index.tsx

```diff
@@ -19,140 +20,138 @@ import { useWithBasePath } from '../lib/useWithBasePath';
 export default function Page() {
 	const withBasePath = useWithBasePath();
 	return (
-		<>
+		<Fragment>
 			<DocumentTitle title="Home" />
-			<SiteLayout>
```

Comment:
All of the documentation site is built using this pattern, e.g. `<SiteLayout />` is part of the Page component.

This causes the Page component to re-render on every route, causing the MainNav component state to rese, which is why we haven't seen the Main nav issue up until now.

Changing this to `Page.getLayout` which is the recommended way from NextJS.